lib/versioning/Version.js

```diff
@@ -64,14 +64,12 @@ class Version {
      * to indicate that the master version of an object has been deleted and it
      * needs to be updated by the latest version if any.
      *
-     * @param {string} versionId - the versionId of the version, this expects
```

Comment:
Note: This might have been a leftover after refactoring - I did not see this param used anywhere in the function, so I removed it from the jsdoc.